tests/src/whisk/core/cli/test/WskEntitlementTests.scala

```diff
@@ -259,6 +265,63 @@ class WskEntitlementTests
             })(defaultWskProps)
     }
 
+    it should "invoke an action sequence from package" in withAssetCleaner(guestWskProps) {
```

Comment:
Not immediately obvious whether we need this test. We have this one: https://github.com/openwhisk/openwhisk/blob/master/tests/src/system/basic/WskSequenceTests.scala#L228

which tests a sequence that points to an action in a package (like the test under review); in addition, the existing test performs checks around parameter passing. Do we need this test you're adding?